### ISSUE TITLE
Fails to generate valid stubs for empty classes

### ISSUE THREAD

**Issue by software-opal** [edreamleo/make-stub-files#2]
An empty class will exist in the generated stub file; but will not have anything as the body of the class meaning the stub is invalid.

Input:
```python
class InvalidTag(Exception):
    pass
```

Output:
```python
class InvalidTag(Exception):
```

Expected Output:
```python
class InvalidTag(Exception): ...
```

**Comment by edreamleo** [comment]
Rev 4bb32f5 is an experimental fix.  Does it work for you?

**Comment by edreamleo** [comment]
Fixed at eea7fbc, with two new unit tests.

Added --silent flag.
Added "tail" logic in st.visit_ClassDef and f.ClassDef.
